src/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+</head>
+<body>
+    <h1>To Do List</h1>
+    <div id="container">
+        <h3>Today´s To Do</h3>
+        <input type="text" name="task" id="new-task" placeholder="Add to your list..." />
+        <div id="list-elem">
```

Comment:
- [ ] Kindly make the project match the given design:

| Original | Yours |
| ----- | ------ |
| <img src="https://camo.githubusercontent.com/d04bee7a5b8f62cc6655dbec65c4d14583e4d78d0c434ddc070f6881bb9568fc/68747470733a2f2f696d672e796f75747562652e636f6d2f76692f416355642d5f596a6a71672f302e6a7067" height="250px"/> | <img src="https://user-images.githubusercontent.com/33323077/134726002-dbc3996b-aaff-433e-820a-5df03424d793.png" height="250px"/> |